sqrt/__init__.py

```diff
@@ -48,6 +49,8 @@ def get_data_stream(iterable):
 
 
 def main(save_to, num_batches, continue_=False):
+    if continue_:
+        continue_training(save_to)
```

Comment:
I'm not sure I follow. My suggestion is just to change `continue_training` to:
```python
    with change_recursion_limit(config.recursion_limit):
        with open(path, "rb") as f:
            main_loop = load(f)
    main_loop.run()
    return main_loop
```
Right now the user passes a path, then in the background it loads a main loop and this main loop object changes, but the user will never have access to it. This means that if you want to do something with the main loop besides calling `.run`, you cannot use `continue_training`. Adding `return main_loop` at the end fixes that, and shouldn't hurt.